src/molecules/SemTabs/SemTabs.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
```

Comment:
I would appreciate if you renamed this to SemesterTabs but you don't have to. Personally I find longer names easier read but that might just be me.